keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -232,20 +232,45 @@ _This section must be completed when targeting alpha to a release._
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes, but CNIs relying on the new field wont recognize it anymore
+  Yes, fields created before feature being disabled will not be changed/dropped 
+  if the old value is kept on the updated object. If the user removes the field 
```

Comment:
It's probably because of my english, but I think I'm not fully parsing this sentence.

Maybe let's say:
`
Yes. One caveat here is that NetworkPolicy objects create with EndPort field set when the
features was enabled will continue to have that field set when the feature is disabled,
but will not have any effect.
`

WDYT?

[i.e. change this whole paragraph to what I wrote]

Reply:
hum, my concern here is that it actually will have effect, if the CNI knows the field.

As an example, let's say Calico is using the last client-go which knows the EndPort field. Once I disable the Feature Gate, per PrepareForCreate and PrepareForUpdate I will not be able to add a new EndPort field (if the EndPort field is not in use in that object), but existing EndPort fields can be changed or removed. So, disabling the FeatureGate only blocks users to add/change on objects that does not have this field, but does not block the CNI to consume this, or even the user to Update or Delete it.

Before reaching your comment here, I've re-written the paragraph as the following:

```
Yes, fields created before feature gate being disabled will not be dropped unless the user removes it from the object.
  If the user removes the field from the object it will be dropped and the 
  field can only be re-inserted if feature gate is enabled again.
```
How does this sounds?